.travis.yml

```diff
@@ -19,6 +19,7 @@ matrix:
         - cargo fmt -- --check
         - cargo build --all-targets
         - cargo test
+        - docker run -it --rm -v $PWD:$PWD -w $PWD -v /var/run/docker.sock:/var/run/docker.sock -v $HOME/.cargo:/root/.cargo rust cargo test
```

Comment:
I feel like this change just made the CI build fail. Maybe it doesn't belong here or maybe I should place it elsewhere.